targets/unity-v2/make.js

```diff
@@ -1,10 +1,10 @@
 var path = require("path");
 
 // Making resharper less noisy - These are defined in Generate.js
-if (typeof (copyTree) === "undefined") copyTree = function () { };
```

Comment:
Sweet! Finally got the chance to update it 👍

Reply:
Yeah, I couldn't find the file in-place, and I realized it was still in templates, so I moved it.